packages/react-server/src/server/server.ts

```diff
@@ -17,10 +17,12 @@ interface Options {
   ip?: string;
   port?: number;
   assetPrefix?: string;
+  proxy?: boolean;
   assetName?: string | ValueFromContext<string>;
   serverMiddleware?: compose.Middleware<Context>[];
   render: RenderFunction;
   renderError?: RenderFunction;
+  app?: Koa;
```

Comment:
I am not sure about exposing app in the API, it doesn't seems to be something we absolutely need other than for test?